### ISSUE TITLE
problem parsing latex equation to pdf or epub files

### ISSUE THREAD

**Issue by klvnsa** [asciidoctor/asciidoctorj#1259]
**MY SETUP**
Windows 10
Java(TM) SE Runtime Environment (build 21.0.1+12-LTS-29)
AsciidoctorJ 2.5.7 (Asciidoctor 2.0.18)
Runtime Environment: jruby 9.3.8.0 (2.6.8)

**MY _test.adoc_ FILE**

```
= My Equation
:stem: latexmath

[stem]
++++
\lim_{n \to \infty}\frac{n}{\sqrt[n]{n!}} = {\large e}
++++
```

**TRIED COMMANDS**
asciidoctorj -b pdf test.adoc  **and**  asciidoctorj -b epub3 test.adoc

**THE PROBLEM**
Running the above commands, the equation doesn't render properly into pdf or epub files.
When I run for html5, it works well. The equation is properly rendered.

What is missing? Could someone help me? Please.

**Comment by slonopotamus** [comment]
I believe you need to use [asciidoctor-mathematical](https://docs.asciidoctor.org/asciidoctor/latest/stem/mathematical/).

**Comment by abelsromero** [comment]
> I believe you need to use [asciidoctor-mathematical](https://docs.asciidoctor.org/asciidoctor/latest/stem/mathematical/).

Yes, PDF rendering has different requirments. @klvnsa you can find more details in https://docs.asciidoctor.org/pdf-converter/latest/stem/

**Comment by klvnsa** [comment]
thanks.
I will use http://chat.asciidoctor.org/
